plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,521 @@
+                    "ledBrightnessLevel": 3,
+                    "ledStatus": "Enabled",
+                    "location": "LTTS",
+                    "macAddress": "90:e9:5e:03:f3:40"
+                }
+            ]
+        Example:
+            functions = Accesspoint(module)
+            ap_data = functions.get_ap_configuration()
+        """
+        ap_config_data = []
+        for device in self.payload["device_list"]:
+            self.log('Getting Access Point Configuration Information for ' \
+                     + device['apEthernetMacAddress'], "INFO")
+            try:
+                jsondata = self.dnac.wireless.get_access_point_configuration(
+                    key=str(device['apEthernetMacAddress']))
+                ap_config_data.append(jsondata)
+            except Exception as e:
+                self.log(jsondata['error'] + e, "ERROR")
+        self.log('Access Point Configuration Information: ' + str(ap_config_data), "INFO")
+        if self.payload["ap_selected_field"] == "" or self.payload["ap_selected_field"] == "all":
+            return ap_config_data
+
+        fields = [str(self.keymap[x]) for x in self.payload["ap_selected_field"].split(",")]
+        if len(ap_config_data) != 0:
+            df = pd.DataFrame.from_records(ap_config_data)
+            selected_data = df[fields]
+            return selected_data.to_dict('records')
+        else:
+            return None
+
+    def compare_ap_cofig_with_inputdata(self, apconfig):
+        """
+        This function used to compare with the input ap detail with the current ap configuration
+        information are not same, those data will be updated in the AP input information.
+        Parameters:
+          - apconfig: This is response of the get_ap_configuration
+        Returns:
+            This will be the return the final data for update AP detail.
+            "final_input": [
+                {
+                    "adminStatus": true,
+                    "apList": [
+                        {
+                            "macAddress": "34:5d:a8:0e:20:b4"
+                        }
+                    ],
+                    "configureLedBrightnessLevel": true,
+                    "ledBrightnessLevel": 4,
+                    "macAddress": "34:5d:a8:0e:20:b4"
+                } ]
+        Example:
+            functions = Accesspoint(module)
+            final_input_data = functions.compare_ap_cofig_with_inputdata(all_apconfig)
+        """
+        final_apchange = []
+        for each_input in self.payload["config"]:
+            for eachap in apconfig:
+                # We are identifing AP based on the AP mac Address so we cannot update this field.
+                if each_input["mac_address"] == eachap["macAddress"]:
+                    newdict = {}
+                    for each_key in list(each_input.keys()):
+                        if each_input[each_key] != eachap[self.keymap[each_key]]:
```

Comment:
Avoid nested for loops and multiple if statements

Reply:
Some scenario we need to use the nested loops so that i have used.  for update the Brightness level to multiple unified AP. we need to use the nested loop.